File: 프로그래머스/level0/로그인 성공.py
def solution(id_pw, db):
    answer = ''
    for i in range(len(db)):
        if id_pw[0] == db[i][0]:
            answer = 'wrong pw'
            break
        else:
            answer = 'fail'
            
    if id_pw in db:
        answer = 'login'
    return answer

File: 프로그래머스/level0/로그인_성공.java
package 프로그래머스.level0;

public class 로그인_성공 {
    public String solution(String[] id_pw, String[][] db) {
        String answer = "fail";
        for (int i = 0; i < db.length; i++) {
            for (int j = 0; j < id_pw.length; j++) {
                if (id_pw[0].equals(db[i][0]) && id_pw[1].equals(db[i][1])) {
                    answer = "login";
                    break;
                }

                if (id_pw[0].equals(db[i][0]) && !id_pw[1].equals(db[i][1])) {
                    answer = "wrong pw";
                }
            }
        }
        return answer;
    }
}
